api/src/main/java/org/pmiops/workbench/leonardo/LeonardoApiClientImpl.java

```diff
@@ -529,13 +530,7 @@ public List<ListPersistentDiskResponse> listPersistentDiskByProjectCreatedByCrea
 
   @Override
   public List<ListPersistentDiskResponse> listDisksAsService() {
-    DisksApi disksApi = serviceDisksApiProvider.get();
-
-    // this call can be slow, so let a long timeout
-    disksApi
-        .getApiClient()
-        .setReadTimeout(workbenchConfigProvider.get().firecloud.lenientTimeoutInSeconds * 1000);
-
+    DisksApi disksApi = withTimeoutInSeconds(120, serviceDisksApiProvider.get());
```

Comment:
here is a way to customize the timeout choice for every call

Reply:
Hm - I'm not quite sure how the Provider logic works:
1. each call to provider.get() initializes a new client, so this change is localized to the listDisks call
2. calling provider.get() returns the existing new client, so it would change all calls

But that's easy to check by debugging.  I'll do that.

Reply:
I confirmed that #1 is true